components/s3_bucket/main.tf

```diff
@@ -27,6 +27,12 @@ variable "roles" {
   default     = []
 }
 
+variable "temporary_paths" {
+  description = "Automatically delete files in the following paths after 14 days."
+  type        = list(string)
+  default     = []
```

Comment:
This says that, by default, we don't have any temporary paths. If we set this variable, it has to be an array of strings.